prisma/schema.sql

```diff
@@ -0,0 +1,158 @@
+CREATE TYPE provinces AS ENUM('NL', 'PE', 'NS', 'NB', 'QC', 'ON', 'MB', 'SK', 'AB', 'BC', 'YT', 'NT', 'NU');
+CREATE TYPE weekdays AS ENUM ('MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN');
+
+-- npx prisma introspect && npx prisma generate
+-- Create users table
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255),
+  email VARCHAR(255) UNIQUE,
+  email_verified TIMESTAMPTZ,
+  image TEXT,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create email verification requests table
+CREATE TABLE verification_requests (
+  id SERIAL PRIMARY KEY NOT NULL,
+  identifier VARCHAR(255) NOT NULL,
+  token VARCHAR(255) NOT NULL UNIQUE,
+  expires TIMESTAMPTZ NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE program (
+  program_id SERIAL PRIMARY KEY NOT NULL,
+  price INTEGER NOT NULL, -- cents
```

Comment:
What does the `-- cents` comment mean?

Reply:
didnt want to deal with floating point, stackoverflow said to use the price in cents and make it an int

Reply:
I see, so $50.00 is stored as 5000 right? Also when we display the price in the frontend, we'll convert it to a more standard format?

Reply:
can we make this comment more descriptive?